pkgs/by-name/sc/scopehal-apps/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  stdenv,
+  breakpointHook,
+  fetchFromGitHub,
+  cmake,
+  git,
+  lib,
+  pkg-config,
+  libsigcxx,
+  gtkmm3,
+  cairomm,
+  yaml-cpp,
+  catch2,
+  glfw,
+  libtirpc,
+  liblxi,
+  glew,
+  libllvm,
+  libdrm,
+  elfutils,
+  libxcb,
+  zstd,
+  libxshmfence,
+  wrapGAppsNoGuiHook,
+  xcbutilkeysyms,
+  systemd,
+  shaderc,
+  vulkan-headers,
+  vulkan-loader,
+  vulkan-tools,
+  glslang,
+  spirv-tools,
+  ffts,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scopehal-apps";
+  version = "e75c0bd1eed4c057bfbbfa6a75a211a88d30c934";
+
+  src = fetchFromGitHub {
+    owner = "ngscopeclient";
+    repo = "scopehal-apps";
+    rev = version;
+    hash = "sha256-sKrxElL4XRHqCPHsH8Q/2pJeQwnMex0YAQMU5I4FEVA=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    wrapGAppsNoGuiHook
+  ];
+
+  buildInputs = [
+    pkg-config
+    libsigcxx
+    gtkmm3
+    cairomm
+    yaml-cpp
+    catch2
+    glfw
+    libtirpc
+    liblxi
+    glew
+    libllvm
+    libdrm
+    elfutils
+    libxcb
+    zstd
+    libxshmfence
+    xcbutilkeysyms
+    systemd
+    vulkan-headers
+    vulkan-loader
+    vulkan-tools
+    spirv-tools
+    glslang
+    shaderc
+    ffts
+  ];
+
+  enableParallelBuilding = true;
```

Comment:
Do we have a rationale for enabling this? Does the compilation takes too long without parallel building?

Reply:
Yes, it is a fairly bulky C++ project. I personally found it very annoying to iterate on this derivation without it.

Reply:
CMake already set `enableParallelBuilding = true` by default so it is not necessary.
This line should be removed and it shouldn't impact build time.